### PR TITLE
[WIP] improve installation guide

### PR DESCRIPTION
**Pull request by nqb**
# Description
- SNMPv3 doc
- Accounting doc

# Impacts
Doc.

# Delete branch after merge
YES

### CODE REVIEW

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 1]
```diff
@@ -2612,6 +2612,13 @@ RADIUS Accounting
 
 RADIUS Accounting is usually used by ISPs to bill clients.  In PacketFence, we are able to use this information to determine if the node is still connected, how much time it has been connected, and how much bandwidth the user consumed.
 
+
+IP log updates
+^^^^^^^^^^^^^^
+
+If you send IP address of nodes in accounting data and want to update iplog entries of your nodes, you can enabled 'Update the iplog using the accounting' setting in 'System configuration -> Main configuration -> Advanced'.
```
Comment (extrafu) [inline docs/PacketFence_Installation_Guide.asciidoc]: If you send the IP address of nodes.... you can enable... setting from ...

**Comment by nqb** [comment]
@extrafu: could you review ?

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 2]
```diff
@@ -4473,6 +4481,20 @@ Here is a switch configuration example in order to enable SNMP v3 in both direct
   snmp-server enable traps port-security trap-rate 1
   snmp-server host 192.168.0.50 version 3 priv readUser  port-security
 
+By default a switch has a SNMPv3 engineID, you can get it with `show snmp engineid`
```
Comment (extrafu) [inline docs/PacketFence_Installation_Guide.asciidoc]: By default a switch has a SNMPv3 engineID => By default a switch has a SNMPv3 engine identifier (SNMPEngineID)

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 3]
```diff
@@ -4473,6 +4481,20 @@ Here is a switch configuration example in order to enable SNMP v3 in both direct
...
+
+Test from a PacketFence server
+++++++++++++++++++++++++++++++
+
+With `net-snmp` package, you can test SNMPv3 communication with your switch:
```
Comment (extrafu) [inline docs/PacketFence_Installation_Guide.asciidoc]: With the `net-snmp` package properly installed, ...

**Pull request merged**